pkg/operator/observe_config.go

```diff
@@ -36,11 +39,13 @@ type ConfigObserver struct {
 	queue workqueue.RateLimitingInterface
 
 	rateLimiter flowcontrol.RateLimiter
+	// observers are used to build the observed configuration
```

Comment:
observers are used to build the observed configuration.  They are called in order and are expected to mutate the config based on cluster state.